### PR TITLE
Error handling for lack of camera

### PR DESCRIPTION
**Pull request by 13ascott**
I added "grab" method to MotionDetection class that utilizes _default_img given the scenario that no eligible camera is detected when opencv class "VideoCapture" is initialized

I also am trying to get familiar with GitHub, please let me know if I'm doing something wrong.

### CODE REVIEW

File: .vs/ProjectSettings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (mlandergan) [inline .vs/ProjectSettings.json]: Please remove this file

File: .vs/PythonSettings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (mlandergan) [inline .vs/PythonSettings.json]: Please remove this file

File: .vs/VSWorkspaceState.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (mlandergan) [inline .vs/VSWorkspaceState.json]: Please remove this file. We don't want these files tracked

File: MotionDetection.py [hunk 4]
```diff
@@ -24,6 +24,14 @@ def __init__(self, display_detections = True):
 		self._current_frame = None
 		self._current_gray_frame = None
 		self._display_detections = display_detections
+		self._default_img = cv2.imread("lib/default_img.png")
```
Comment (mlandergan) [inline MotionDetection.py:27]: is this a default image included in cv2 or do we need to push a lib folder with this image?

**Review by mlandergan — CHANGES_REQUESTED** [review]

File: MotionDetection.py (hunk 4 above)
Comment (13ascott) [reply MotionDetection.py:27]: This is an image that we would include in the "lib" folder.
If this is something you want to get away from, we can just create a basic "default_img" using some of the opencv GUI tools.

Comment (13ascott) [reply MotionDetection.py:27]: I did include both default_img.png and the lib folder in this commit

**Comment by mlandergan** [comment]
Great work, thanks Austin. Can you squash your commits into a single commit and then I'll merge it.

**Comment by 13ascott** [comment]
I figured out the squashing... just took a force push to the origin.
Let me know if I need to do anything else

**Pull request merged**